.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
I wonder if we need a comment in here (and in ansible-test's config) to remind people to keep these two files in sync

Reply:
yeah, I think it'd be helpful not now but in the future. Honestly I'd love for pep8 to be able to map its ignores to a list, pretty much like a `requirements.txt`